backend/src/test/java/net/vidux/camhub/discovery/CameraScanTest.java

```diff
@@ -0,0 +1,61 @@
+package net.vidux.camhub.discovery;
+
+import java.util.Set;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.MethodSource;
+
+class CameraScanTest {
```

Comment:
Hol vannak a tesztek, amik a CameraScan osztályt tesztelik?

Reply:
Ehhez annyit tennék hozzá, hogy konkrét implementációkat tesztelünk, tehát a teszt osztály neve annak az osztálynak a nevével kellene egyeznie, amelyik osztályt teszteled. 

Pl.: 

- CameraScanViduxHelper -> CameraScanViduxHelperTest, 
- RawCameraDataFactory -> RawCameraDataFactoryTest, 
- StreamGobbler -> StreamGobblerTest, 
- ViduxHelperRunner -> ViduxHelperRunnerTest

Tesztek írása közben érdemes a teszt lefedettséget is nézni és lehetőleg úgy min. 80%-os lefedettséget érdemes elérni osztályonként.